docs/content/Cube-Cloud/Developer-Tools/Alerts.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Alerts
+permalink: /cloud/dev-tools/alerts
+category: Developer Tools
+menuOrder: 3
+---
+
+Setting up alerts in Cube Cloud is easy - click your username in the top-right
+for the menu, then click Alerts:
+
+<div style="text-align: center">
+  <img
+    alt="Finding the Alerts menu in Cube Cloud"
+    src="./alerts-1.png"
+    style="border: none"
+    width="100%"
+  />
+</div>
+
+You should now see the Alerts page - the example screenshot below shows no
+alerts configured for this deployment:
+
+<div style="text-align: center">
+  <img
+    alt="The Cube Cloud account's Alerts Page"
+    src="./alerts-2.png"
+    style="border: none"
+    width="100%"
+  />
+</div>
+
+Click the 'New Alert' button to bring up a dialog where you can configure the
+alert:
+
+<div style="text-align: center">
+  <img
+    alt="Creating a new Alert on Cube Cloud"
+    src="./alerts-3.png"
+    style="border: none"
+    width="100%"
+  />
+</div>
+
+## Alert Types
+
+Cube Cloud lets you set up alerts so you can be notified when there are issues
+with any of the deployments in your account. There are three different types of
+alerts:
+
+- API outages: These are triggered if the REST/GraphQL/SQL APIs become
```

Comment:
were you able to verify that? I have a feeling it actually might work for REST endpoint only, or REST and GraphQL

Reply:
I wasn't able to verify, was hoping you'd confirm 😀